PLIER_pathways_analysis.Rmd

```diff
@@ -0,0 +1,76 @@
+---
```

Comment:
This filename should probably get a number. I assume it would be `8-PLIER_pathways_analysis.Rmd`.

Reply:
It also makes sense to me to include running this notebook in `run_all_analyses_and_plots.sh`, for example:

```sh
Rscript -e "rmarkdown::render('8-PLIER_pathways_analysis.Rmd', clean = TRUE)"
```

We'd have to think about the logic a bit, but we could always use [`params`](https://bookdown.org/yihui/rmarkdown/params-knit.html) here. At the very least, the README should get updated to add this notebook (maybe under individual analyses?)

Reply:
Changed file name to `8-PLIER_pathways_analysis.Rmd` and added it to `run_machine_learning_experiments.sh`

Reply:
👍 yea that works! thank you